sys/net/application_layer/dhcpv6/client.c

```diff
@@ -247,8 +255,9 @@ static inline size_t _compose_elapsed_time_opt(dhcpv6_opt_elapsed_time_t *time)
     return len + sizeof(dhcpv6_opt_t);
 }
 
+#ifdef MODULE_GNRC_DHCPV6_CLIENT_MUD_URL
 static inline size_t _compose_mud_url_opt(dhcpv6_opt_mud_url_t *mud_url_opt,
-                                          const char *mud_url, size_t len_max)
+                                          size_t len_max)
 {
```

Comment:
You can save an `#ifdef` down the line if this just returns 0 if the module is not used.

```suggestion
{
#ifndef MODULE_GNRC_DHCPV6_CLIENT_MUD_URL
    return 0;
#else
```

Reply:
Thanks for the suggestion! I added it down below :)